rfc/src/rfcs/0013-list.md

```diff
@@ -20,53 +20,51 @@ This feature will not cause any regressions for exisiting users.
-
-Contract Harnesses:
-- example::verify::check_foo_u32
-- example::verify::check_foo_u64
-- example::verify::check_func
-- example::verify::check_bar
-
 Contracts:
-|--------------------------|-----------------------------------------------|
-| Function                 |  Contract Harnesses                           |
-|--------------------------|-----------------------------------------------|
-| example::impl::func      |  example::verify::check_func                  |
-|--------------------------|-----------------------------------------------|
-| example::impl::bar       |  example::verify::check_bar                   |
-|--------------------------|-----------------------------------------------|
-| example::impl::foo       |  example::verify::check_foo_u32,              |
-|                          |  example::verify::check_foo_u64               |
-|--------------------------|-----------------------------------------------|
-| example::prep::parse     |  NONE                                         |
-|--------------------------|-----------------------------------------------|
-
-Totals:
-- Standard Harnesses: 2
-- Contract Harnesses: 4
-- Functions with Contracts: 4
-- Contracts: 10
+|-------|-------------------------|----------------|--------------------------------------------------|
+|       | Function                | # of Contracts | Contract Harnesses (#[kani::proof_for_contract]) |
+|-------|-------------------------|----------------|--------------------------------------------------|
+|       | example::impl::bar      | 4              | example::verify::check_bar                       |
+|-------|-------------------------|----------------|--------------------------------------------------|
+|       | example::impl::baz      | 0              | example::verify::check_baz                       |
+|-------|-------------------------|----------------|--------------------------------------------------|
+|       | example::impl::foo      | 2              | example::verify::check_foo_u32                   |
+|       |                         |                | example::verify::check_foo_u64                   |
+|-------|-------------------------|----------------|--------------------------------------------------|
+|       | example::impl::func     | 1              | example::verify::check_func                      |
+|-------|-------------------------|----------------|--------------------------------------------------|
+|       | example::prep::parse    | 1              | NONE                                             |
+|-------|-------------------------|----------------|--------------------------------------------------|
+| Total | 5                       | 8              | 5                                                |
+|-------|-------------------------|----------------|--------------------------------------------------|
+
+Standard Harnesses (#[kani::proof]):
+1. example::verify::check_modify
+2. example::verify::check_new
```

Comment:
Why a numbered list?

Reply:
Because it's a concise way to see how many standard harnesses you have--we can report the list and the total simultaneously.

Reply:
I personally like the previous format better with the explicit total. But I'll leave it up to you. Thanks